src/Types/Product.php

```diff
@@ -148,6 +148,8 @@ class Product extends AbstractType
 
     public const ARROW_SUBCATEGORY = 'arrowSubCategories';
 
+    public const INDIRECT_BUSINESS = 'indirectBusiness';
```

Comment:
Please add your new field in ProductTest.php
Please also document your change in CHANGELOG.md

Reply:
yes I was on Unit Test, and now i changed the changelog